modules/order/src/Adjustment.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Drupal\commerce_order;
```

Comment:
Time to create a commerce_pricing module? We'll want pricing to work even without orders?